webapp/src/simulator.ts

```diff
@@ -66,6 +69,28 @@ export function init(root: HTMLElement, cfg: SimulatorConfig) {
         }
     }
 
+    // Map simulator extensions from approved repos
+    const simulatorExtensions: pxt.Map<pxt.SimulatorExtensionConfig> = {};
+    Object.entries(trgConfig?.packages?.approvedRepoLib || {})
+        .map(([k, v]) => ({ k: k, v: v.simx }))
+        .filter(e => !!e.v)
```

Comment:
What does the `e` stand for here?

Reply:
It could stand for `element`, I suppose. There are other names that would have worked equally well (e.g.: `v`). In short callbacks like this where the meaning of the parameter is obvious, concise code tends to be clearer and more readable. It probably has historical origin in for loop index naming: `i`, `x`, etc. There's a practicality to using single-character names as long as the callback method itself is very simple. If the function is complex, spanning many lines and containing conditional code blocks, then descriptive naming would be preferred.

Was something unclear about the use of `e` here?

Reply:
I think just all of the single-letter variables are a kinda tricky to follow. I also usually associate `e` with events, so that's what I originally thought it was. Could this maybe be `el` instead?

Reply:
I'd like to stick with one-letter names in the single-line handlers. How about `v`?

Reply:
Hm, `v` doesn't work well due to `v.v`. I'll go with `el`. `el.k` and `el.v` are both descriptive and a little amusing.

Reply:
Ha, instead of a bug in the code, we have an elk in our code!

Reply:
`el` doesn't actually work well either due to it being used in many places elsewhere in the file to refer to a document element. Will try `x` I guess.